cmd/lncli/commands.go

```diff
@@ -146,6 +146,52 @@ func newAddress(ctx *cli.Context) error {
 	return nil
 }
 
+var estimateFeeCommand = cli.Command{
+	Name:      "estimatefee",
+	Category:  "On-chain",
```

Comment:
This should be lower case?

Reply:
`"On-chain"` is what is used throughout the file.